packages/eslint-config-react-native/index.js

```diff
@@ -1,119 +1,16 @@
-    // and accessibilityComponentType props set.
-    // <Touchable*> components must have the accessibilityRole prop or both accessibilityTraits and
-    // accessibilityComponentType props in order to be fully accessible.
-    'react-native-a11y/has-accessibility-props': 'error',
-
-    'react-native-a11y/has-accessibility-hint': 'error',
-
-    'react-native-a11y/has-valid-accessibility-actions': 'error',
-
-    // Enforce accessibilityComponentType property value is valid.
-    // The accessibilityComponentType property is essentially the android version of
-    // accessibilityTraits and is used to alert a user, using TalkBack, what kind of element they
-    // have selected.
-    'react-native-a11y/has-valid-accessibility-component-type': 'error',
-
-    'react-native-a11y/has-valid-accessibility-ignores-invert-colors': 'error',
-
-    // Enforce accessibilityLiveRegion prop values must be valid.
-    // On android devices, when components dynamically change, we want TalkBack to alert the end
-    // user. This is made possible by the accessibilityLiveRegion property.
-    'react-native-a11y/has-valid-accessibility-live-region': 'error',
-
-    // Enforce accessibilityRole property value is valid.
-    // The accessibilityRole property is used to tell Talkback or Voiceover the role of a UI
-    // Element.
-    'react-native-a11y/has-valid-accessibility-role': 'error',
-
-    // Enforce accessibilityStates property value is valid.
-    // The accessibilityStates property is used to tell Talkback or Voiceover the state a UI Element
-    // is in.
-    'react-native-a11y/has-valid-accessibility-states': 'error',
-
-    // Enforce accessibilityTraits and accessibilityComponentType prop values must be valid.
-    // The accessibilityTraits property is used to tell a person using VoiceOver what kind of
-    // element they have selected.
-    'react-native-a11y/has-valid-accessibility-traits': 'error',
-
-    'react-native-a11y/has-valid-accessibility-value': 'error',
-
-    // Enforce importantForAccessibility property value is valid.
-    // If there are two (or more) overlapping UI components with the same parent, default
-    // accessibility focus can have unpredictable behavior. The importantForAccessibility property
-    // will resolve this by controlling if a view fires accessibility events and if it is reported
-    // to accessibility services.
-    'react-native-a11y/has-valid-important-for-accessibility': 'error',
-
-    // Enforce if a view has accessible={true}, that there are no touchable elements inside.
-    // <Touchable*> or will not work inside an accessible element. Any element that has the
-    // accessible={true} property (along with the accessibleLabel property) must therefore not
-    // contain any <Touchable*> or elements.
-    'react-native-a11y/no-nested-touchables': 'error',
+    'react-hooks/exhaustive-deps': 'error',
```

Comment:
Can we please add a description/comment to every single rule we are adding/changing here?